### PR TITLE
Add translation for Russian language

### PR DESCRIPTION
**Pull request by Kanaduchi**

### CODE REVIEW

File: locales/ru/config.json [hunk 1]
```diff
@@ -0,0 +1,82 @@
+{
```
Comment (laymonage) [inline locales/ru/config.json:1]: Could you also translate this file?

File: locales/ru/common.json [hunk 2]
```diff
@@ -0,0 +1,102 @@
+{
+  "reactions": {
+    "0": "{{count}} откликов",
+    "one": "{{count}} отклик",
+    "few": "{{count}} отклика",
+    "other": "{{count}} откликов"
```
[the remark this answers is not in the document]
Comment (Kanaduchi) [reply locales/ru/common.json]: Fixed

File: locales/ru/config.json (hunk 1 above)
Comment (Kanaduchi) [reply locales/ru/config.json:1]: Fixed

**Comment by laymonage** [comment]
Спасибо!

**Pull request merged**